resources/js/admin/auth-clients/components/AuthClientsListing.vue

```diff
@@ -139,6 +139,10 @@ export default {
 
       //Pagination
       let meta = {};
+      if (this.perPage >= data.length) {
```

Comment:
Doesn't the pagination come down as part of the axios returned object? Why don't we just reference it?

![image](https://user-images.githubusercontent.com/535300/83809163-dc3dc000-a66a-11ea-8b2e-ba1ec85aecd3.png)

Reply:
the api is from laravel Passport and does not contain the required meta data.